zproject/prod_settings_template.py

```diff
@@ -2,9 +2,8 @@
 
 from .config import get_secret
 
-
```

Comment:
This newline should be deleted in the first commit rather than the fourth.

Reply:
Fixed.